File: statistic.R
library(ggplot2)
library(jsonlite)
data <- fromJSON("D:\\Code Library\\foody_servey\\result--top500.json")
tmp <- data.frame(y = data$key, x = data$value)
tmp
p <- ggplot(data=tmp, aes(x=y, y=x)) +
  labs(x="District", y="Number of Shop", title="Number of Cafe Shop") +
  geom_bar(stat="identity") + theme_bw()

p

File: statistic.js
const fs = require('fs-extra');
const util = require('util');


['', 'coffee', 'cafe'].forEach(q => {
    var filename = './result-format' + (q !== "" ? "-" + q : "") + '.json';
    console.log(filename);
    var data = fs.readJSONSync(filename);
    var total = 0;
    var filters = [];
    try {
		let tmp = [];
        data.forEach(it => {
            total += it.value;
            if (it.value >= 700) {
                filters.push(util.format('%s: %d', it.key, it.value));
				tmp.push({ 'key': it.key, 'value': it.value });
            }
        })

        console.log("=========== %s", q)
        console.log("Total: %d cửa hàng", total);
        console.log(filters);
		fs.outputFileSync(`./result-${q}-top500.json`, JSON.stringify(tmp, null, 4));
    } catch (e) {
        console.error(e);
    }
})